api/test/unit/utils/throttle_test.py

```diff
@@ -129,8 +130,11 @@ def test_abstract_oauth2_id_rate_throttle_does_not_apply_if_token_app_rate_limit
 
 @pytest.mark.django_db
 def test_rate_limit_headers(request_factory):
+    cache.clear()  # This is needed between multiple runs on the same computer.
```

Comment:
This will destroy any local data used in redis cache which could interrupt someone doing other testing if they chose to run the entire test suite. Can we use the mock redis fixture for this instead? That way this test doesn't destroy the local environment's data.

Reply:
The `redis` fixture at the top of the file has `autouse=True`. I think it's using that fixture by default.

Reply:
Changed the line to only remove caches pertaining to throttling in a186341.